pkg/openstack/glance.go

```diff
@@ -55,7 +59,7 @@ func ReconcileGlance(ctx context.Context, instance *corev1beta1.OpenStackControl
 			if glanceAPI.Override.Service == nil {
 				glanceAPI.Override.Service = map[service.Endpoint]service.RoutedOverrideSpec{}
 			}
-			glanceAPI.Override.Service[endpointType] = AddServiceComponentLabel(
+			glanceAPI.Override.Service[endpointType] = AddServiceOpenStackOperatorLabel(
```

Comment:
yes, thats an issue? we can also use a different selector key

Reply:
it shouldn't be an issue. From a user perspective we used to `oc get ..` all the resources associated to glance using `-l service=glance`, and looking at this part I thought that this key would have been overwritten by the openstack-operator.
We still have different strategies to get that view, so it's really a minor thing considered the problem you're trying to solve here. I'm wondering if it's the same for the other services.

Reply:
I have updated the PR so that the openstack-operator will use its own service selector and not conflict with any common.AppSelector usage inside the service operators

Reply:
+1 from me, that's better I guess